index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alazar Esheetu</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  </head>
+  <body>
+    <div class="contanier">
+      <nav class="nav-contanier">
+        <div class="logo">Alazar Eshetu</div>
+        <div>
+          <img class="menu-icon" src="Images/Icon - Menu.png" alt="menu-icon" />
+        </div>
+      </nav>
+      <div class="headline">
+        <div><h1>Hey There. I’m Alazar</h1></div>
+        <div><p class="head">I’am a Software Developer</p></div>
+        <div>
+          <p class="description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="link">
+          <div class="all raddit">
+            <img src="Images/raddit.png" alt="raddit-icon" />
+          </div>
+          <div class="all Linkdn">
+            <img src="Images/Linkdn.png" alt="linkedn-icon" />
+          </div>
+          <div class="all peace"><img src="Images/peace.png" alt="Icon" /></div>
+          <div class="all tewtter">
+            <img src="Images/twet.png" alt="tewtter-icon" />
+          </div>

```

Comment:
- [ ] Great job on overall design :1st_place_medal: In some of your designs some issues to figure out. Kindly wrap the social media image icons in unorder list `ul and li` instead of `div` for semantic and web accessibility reasons. :+1: 

|Template design|Your design|
|----------------------|----------------|
| ![Screenshot from 2022-12-01 17-59-58](https://user-images.githubusercontent.com/65129228/206218108-65cb8ac8-a11c-4e1e-8362-85c47aee7299.png) | ![Screenshot from 2022-12-07 17-42-30](https://user-images.githubusercontent.com/65129228/206218312-cf1ff754-5c5c-4434-8a34-11ce36148c68.png) |